addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -86,41 +86,59 @@ If the things are not defined in **.things** files the first time the tracker se
 These dynamic channels require the following parameters:
 
 | Parameter | Type |Description |
 | --- | --- | --- |
 | Region Name | String | Region name. If the region is configured in the tracker app as well use the same name. Distance channels can also be defined as binding only regions (not configured in trackers) | 
 | Region center | Location | Region center location |
+| Region Radius | Integer | Geofence radius |
 
-Distance values will be updated each time a GPS log record is received from the tracker.
+Distance values will be updated each time a GPS location log record is received from the tracker. 
 
-#### Geofences
+When this calculated distance is less than the defined geofence radius the binding also fires event on Region Trigger channel. 
 
-Switch type items can be linked to **regionTrigger** and **regionDistance** channels with the following parameters:
+* When the tracker is approaching (the new calculated distance is less then the previous one) the payload is <<region_name>>/enter. 
+* If the tracker is distancing (the new calculated distance is greater then the previous one) payload is <<region_name>>/leave. 
 
-| Channel | Parameter | Type |Description |
-| --- | --- | --- | --- |
-| Region Trigger | Region Name | String | Region name which should be the same as used in the tracker application |
-| Region Distance | Region Circle Radius | Number | Region radius |
+If the tracker is moving inside/outside the region (both the previous and the current calculated distance value is less/greater than the radius) no events are fired. 
+This means that the region events are triggered **ONLY IN CASE THE REGION BORDER IS CROSSED**.
 
-Each time the **Region Trigger** channel is triggered by an entering transition message the binding turns ON the linked switch and turns if OFF in case the region is left.
-For distance channels if the distance is less than the preset radius for the link the binding turns ON the switch and turns it OFF otherwise.
+**Note**: In case the location is set for openHAB installation (Configuration/System/Regional Settings) the binding automatically creates the System Distance channel (gpstracker:tracker:??:distanceSystem) with region name set to **System**.
+
+#### Tracker Location Status
+
+By default the binding notifies the openHAB about entering/leaving event through region trigger channel. 
+In order to have this state available in stateful switch items (e.g. for rule logic) switch type items can be linked to **regionTrigger** channel. 
+There is a special profile (gpstracker:trigger-geofence) that transforms trigger enter/leave events to switch states:
+
+* **<<region_name>>/enter** will update the switch state to **ON**
+* **<<region_name>>/leave** will update the switch state to **OFF**
+
+To link a switch item to regionTrigger channel the following parameters are required by the item link:
+
+| Parameter | Type |Description |
```

Comment:
Please apply the Markdown Table formatter: https://www.tablesgenerator.com/markdown_tables

Reply:
Ok